src/containers/PsychoAssess/sections/Questions.jsx

```diff
@@ -1,16 +1,119 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
-import { Col, Row, Button, Form } from "react-bootstrap";
+import { Col, Row, Button, Modal } from "react-bootstrap";
+import { Link } from "react-router-dom";
 import "../index.css";
-import { useState } from "react";
 
 const OPTION_QUANTITY = 4;
 const QUESTION_QUANTITY = 13;
 
 export default function Questions() {
-  const [isSubmitted, setSubmitted] = useState(false);
+  const [submitModalShow, setSubmitModalShow] = React.useState(false);
   const { t } = useTranslation();
 
+  const SubmitModal = (props) => {
+    return (
+      <Modal
+        {...props}
+        size="md"
+        aria-labelledby="contained-modal-title-vcenter"
+        centered
+      >
+        <Modal.Header closeButton>
+          <Modal.Title id="contained-modal-title-vcenter">
+            {t("assessment.title")}
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>{showResults(calculateScore())}</Modal.Body>
+        <Modal.Footer>
+          <Link to="/contact">
+            <Button className="testContactButton" variant="primary">
+              {t("assessment.contactButton")}
+            </Button>
+          </Link>
+          <Button className="testDoneButton" onClick={props.onHide}>
+            {t("assessment.closeButton")}
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  };
+
+  const showResults = (score) => {
+    if (score !== -15) {
+      if (score >= 52) {
```

Comment:
there should be 0 magic numbers in this function
```
if (score >= Scores.VERY_SEVERE) {
} else if (score >= Scores.SOME_OTHER_TIER) {
} else if (...)
```

Reply:
done, please take a look when available.

Reply:
done, please take a look when available.